Inc/HALALMock/Services/Communication/Ethernet/TCP/Socket.hpp

```diff
@@ -27,11 +36,12 @@ class Socket : public OrderProtocol{
 	uint32_t local_port;
 	IPV4 remote_ip;
 	uint32_t remote_port;
-	tcp_pcb* connection_control_block;
-	tcp_pcb* socket_control_block;
 	SocketState state;
-	queue<struct pbuf*> tx_packet_buffer;
-	queue<struct pbuf*> rx_packet_buffer;
+	//Change the data type of tx/rx packet_buffer
+	queue<vector<uint8_t>> tx_packet_buffer;
+	queue<vector<uint8_t>> rx_packet_buffer;
```

Comment:
probably just a vector of `Packet*` not of `vector<uint8_t>`

Reply:
Do you thing is necesarry to change it due to I think now the implementacion with vetor<uitnt8> could work.